packages/yoastseo/src/languageProcessing/languages/ja/helpers/getStemmer.js

```diff
@@ -1,21 +1,11 @@
-import { get } from "lodash-es";
 import { languageProcessing } from "yoastseo";
 const { baseStemmer } = languageProcessing;
-import createWordForms from "./internal/createWordForms";
 
 /**
  * Returns the stemmer for a researcher.
  *
- * @param {Researcher} researcher The researcher.
- *
  * @returns {Function} The stemmer.
  */
-export default function getStemmer( researcher ) {
-	const morphologyData = get( researcher.getData( "morphology" ), "ja", false );
-
-	if ( morphologyData ) {
-		return word => createWordForms( word, morphologyData );
-	}
-
+export default function getStemmer() {
```

Comment:
Have you checked where `getStemmer` is used? What are the consequences of always returning the base stemmer?

In the [getInternalLinking](https://github.com/Yoast/wordpress-seo/pull/17565/files) PR, a `customGetStemmer` is introduced. Should we make that the general `getStemmer` for Japanese after it's been implemented?